week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word;
 }
```

Comment:
What you have been asked here is to use a string method which deletes the space between word. Search that by typing String methods and use it here.